pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -134,6 +134,10 @@ var removedPluginsByVersion = []removedPlugins{
 		schemeGroupVersion: v1beta3.SchemeGroupVersion.String(),
 		plugins:            []string{},
 	},
+	{
+		schemeGroupVersion: v1.SchemeGroupVersion.String(),
```

Comment:
maybe worth removing this logic as we don't plan to remove plugins past v1.

But you could do it as a separate PR (it might make more sense to submit it before this PR).

Reply:
/hold
for #108444

Otherwise nothing is preventing `SelectorSpread` to be used in v1. Also, we need to update the KEP graduation criteria.

Reply:
`SelectorSpread` plugin is not default enabled, but can still be reenabled by KubeSchedulerConfiguration, we shouldn't prevent it from being used until we remove it from in-tree plugins totally.

Reply:
We should prevent it from being used. That's what we did from v1beta1 to v1beta2: we disallowed some plugins.